.github/workflows/release.yml

```diff
@@ -10,7 +10,33 @@ env:
 jobs:
   build:
     runs-on: ubuntu-latest
+
     steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Build
+        run: ./build.sh
+
+      - name: Service and enclave artifacts
```

Comment:
Can you remove this step from the `build` workflow. It's not really needed now.

Reply:
Do you mean from the new 'pr' one? (You are commenting the 'release.yaml' file).

I don't mind removing it.

Reply:
Yes, from the new pr.yml

Reply:
OK. Check again if all steps are fine.